tlsutil/config_test.go

```diff
@@ -494,3 +509,26 @@ func TestConfig_IncomingTLS_TLSMinVersion(t *testing.T) {
 		}
 	}
 }
+
+func TestConfig_ParseCiphers(t *testing.T) {
+	testOk := "TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_256_CBC_SHA,TLS_RSA_WITH_AES_256_GCM_SHA384"
+	v, err := ParseCiphers(testOk)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if len(v) != 12 {
```

Comment:
If you compare the values instead of the lengths you get a more robust test.

```go
ciphers := []uint16{...}
if got, want := v, ciphers; !reflect.DeepEqual(got, want) {
    t.Fatalf("got ciphers %#v want %#v", got, want)
}
```